doc/manual/parallel-computing.rst

```diff
@@ -849,24 +850,21 @@ object (with appropriate fields initialized) to ``launched`` ::
 Most of the fields in :class:`WorkerConfig` are used by the inbuilt managers.
 Custom cluster managers would typically specify only ``io`` or ``host`` / ``port``:
 
-If ``io`` is specified, it is used to read host/port information. A Julia
-worker prints out its bind address and port at startup. This allows Julia
-workers to listen on any free port available instead of requiring worker ports
-to be configured manually.
-
-If ``io`` is not specified, ``host`` and ``port`` are used to connect.
-
-``count``, ``exename`` and ``exeflags`` are relevant for launching additional workers from a worker.
-For example, a cluster manager may launch a single worker per node, and use that to launch
-additional workers. ``count`` with an integer value ``n`` will launch a total of ``n`` workers,
-while a value of ``:auto`` will launch as many workers as cores on that machine.
-``exename`` is the name of the ``julia`` executable including the full path.
-``exeflags`` should be set to the required command line arguments for new workers.
-
-``tunnel``, ``bind_addr``, ``sshflags`` and ``max_parallel`` are used when a ssh tunnel is
-required to connect to the workers from the master process.
-
-``userdata`` is provided for custom cluster managers to store their own worker specific information.
+- If ``io`` is specified, it is used to read host/port information. A Julia
+  worker prints out its bind address and port at startup. This allows Julia
+  workers to listen on any free port available instead of requiring worker ports
+  to be configured manually.
+- If ``io`` is not specified, ``host`` and ``port`` are used to connect.
+- ``count``, ``exename`` and ``exeflags`` are relevant for launching additional workers from a worker.
+  For example, a cluster manager may launch a single worker per node, and use that to launch
+  additional workers.
+    - ``count`` with an integer value ``n`` will launch a total of ``n`` workers.
```

Comment:
something about this is invalid rst `doc/manual/parallel-computing.rst:861: ERROR: Unexpected indentation.`